### PR TITLE
Add gradient clipping to PPO.

### PR DESCRIPTION
**Pull request by kevinzakka**

### CODE REVIEW

File: brax/training/agents/ppo/train.py [hunk 1]
```diff
@@ -241,7 +243,13 @@ def train(
       preprocess_observations_fn=normalize)
   make_policy = ppo_networks.make_inference_fn(ppo_network)
 
-  optimizer = optax.adam(learning_rate=learning_rate)
+  if max_grad_norm is not None:
```
Comment (btaba) [inline brax/training/agents/ppo/train.py:246]: Can you add a TODO to move gradient clipping to brax/training/gradients.py ?

**Review by btaba — CHANGES_REQUESTED** [review]
